app/src/datasets/odrive/file_conversion.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pandas as pd
+import math
+import datetime
+
+names_to_replace = {
+    "CLIO IV": "CLIO 4",
+    "CLIO III": "CLIO 3",
+    "MÉGANE": "MEGANE",
+    "ZOÉ": "ZOE",
+}
+
+
+def clean_modele(modele):
+    modele = str(modele).upper()
+    if modele in names_to_replace.keys():
+        modele = names_to_replace.get(modele)
+    return modele
+
+
+def clean_date(date_value):
+    if date_value == math.nan or date_value == "00/01/00" or date_value is None:
+        date_value = datetime.datetime.today()
+    return date_value
+
+
+def main():
+    data_xls = pd.read_excel("C:/Users/Artus/batch7_beges/data/raw/odrive/odrive.xlsx", index_col=None)
+    data_xls["Modèle"] = list(map(clean_modele, data_xls["Modèle"]))
```

Comment:
I think you could leverage the pandas api using `apply`:
```suggestion
    data_xls["Modèle"] = data_xls["Modèle"].apply(clean_modele)
```
You might also go a bit further I think :
```suggestion
    data_xls["Modèle"] = data_xls["Modèle"].str.upper()
    data_xls["Modèle"].replace(names_to_replace, inplace=True)
```